sydent/threepid/bind.py

```diff
@@ -45,7 +46,21 @@ def addBinding(self, valSessionId, clientSecret, mxid):
 
         self.sydent.pusher.doLocalPush()
 
+        joinTokenStore = JoinTokenStore(self.sydent)
+        pendingJoinTokens = joinTokenStore.getTokens(s.medium, s.address)
+        invites = []
+        for token in pendingJoinTokens:
+            token["mxid"] = mxid
+            token["signed"] = {
+                "mxid": mxid,
+                "token": token["token"],
+            }
+            token["signed"] = syutil.crypto.jsonsign.sign_json(token["signed"], self.sydent.server_name, self.sydent.keyring.ed25519)
+            invites.append(token)
+        if invites:
+            assoc["invites"] = invites
+            joinTokenStore.deleteTokens(s.medium, s.address)
+
```

Comment:
Does this go wrong if the response to the bind request goes missing, in that if the client retries, the join tokens will have been deleted?